tests/test_prompt.py

```diff
@@ -247,3 +256,32 @@ def test_should_read_userchoice(self, mocker, choices, context):
         )
         read_choice.assert_called_once_with('orientation', choices)
         assert expected_choice == actual_choice
+
+    def test_should_read_yes_no(self, mocker, bool_context):
+        key = 'is_alive'
+        expected_bool = False
+
+        read_bool = mocker.patch('cookiecutter.prompt.read_user_yes_no')
```

Comment:
It would be great to also test that neither ``prompt_choice_for_config`` nor ``read_user_variable`` are called for **bool** variables. :smile: